Dockerfile

```diff
@@ -27,6 +27,7 @@ COPY . fineract
 
 WORKDIR fineract
 RUN sed -i 's/localhost/fineractmysql/' ./fineract-provider/build.gradle
```

Comment:
> Eventually, this will be all that we need to do, rather than updating the build.gradle file.

only after https://issues.apache.org/jira/browse/FINERACT-765, or could this be done now already?